src/components/VerifiedBadge.js

```diff
@@ -1,20 +1,15 @@
+import React from 'react';
 import CheckIcon from '@mui/icons-material/Check';
 import { Chip } from '@mui/material';
-import { makeStyles } from 'models/makeStyles';
-import React from 'react';
-
-const useStyles = makeStyles()(() => ({
-  chip: {
-    borderRadius: '4px',
-  },
-}));
+import { useTheme } from '@mui/material/styles';
 
 function VerifiedBadge({ verified, badgeName }) {
-  const { classes } = useStyles();
+  const theme = useTheme();
+
   return (
     <Chip
-      className={classes.chip}
-      color={!verified ? 'primary' : 'secondary'}
+      sx={{ borderRadius: theme.spacing(2), marginRight: theme.spacing(2) }}
```

Comment:
`marginRight: theme.spacing(2)` can be shortened to `mr: 2` when using the sx prop. Here are some other shortcuts available https://mui.com/system/the-sx-prop/

Positioning styles like `marginRight` should usually not be included in components that are expected to be reused like this `VerifiedBadge`.

Reply:
In short, I love the sx and now it's the most case (in my cases) that I compose CSS

Reply:
Thanks for the tip @QuaidBartolomei, good to know. I updated the code accordingly.